lib/runner.js

```diff
@@ -53,14 +53,25 @@ Runner.prototype.addSerialTest = function (title, cb) {
 };
 
 Runner.prototype.addBeforeHook = function (title, cb) {
-	this.tests.before.push(new Test(title, cb));
+	var test = new Test(title, cb);
+	test.type = 'hook';
```

Comment:
How about moving types to something like `new Test.hook(title, cb)` (can be done in separate PR)?